src/main/java/com/commercetools/service/paypalPlus/impl/BasePaypalPlusService.java

```diff
@@ -51,15 +53,16 @@ protected Logger getLogger() {
     protected final <R> CompletionStage<R> paypalPlusStageWrapper(
             @Nonnull PayPalRESTExceptionSupplier<APIContext, R> supplier)
             throws PaypalPlusServiceException {
+        ExtendedAPIContext extendedAPIContext = paypalPlusApiContextFactory.createAPIContext();
```

Comment:
Before this statement was with try catch block now eventual exceptions will not be catched anymore? Was it intentional?

Reply:
`paypalPlusApiContextFactory.createAPIContext()` actually does not through any exception. It will basically create an instance from the factory. The potential error can happen when it calls the supplier function with the `ExtendedAPIContext` object.